data-plane/pom.xml

```diff
@@ -60,6 +60,7 @@
     <kafka.version>3.2.3</kafka.version>
     <debezium.version>1.9.6.Final</debezium.version>
     <jib.version>3.2.1</jib.version>
+    <quarkus.version>2.12.2.Final</quarkus.version>
```

Comment:
TODO: bump this to https://mvnrepository.com/artifact/io.quarkus/quarkus-bom/2.13.2.Final